File: BirthdayCakeCandles.java
import java.util.Scanner;

import java.util.List;
import java.util.Collections;
import java.util.stream.Stream;
import java.util.stream.Collectors;

public class BirthdayCakeCandles {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        System.out.print("Insert the number of candles: ");
        int n = in.nextInt();
        in.nextLine();

        System.out.print("Insert the candles: ");
        List<Integer> candles = Stream.of(in.nextLine().replaceAll("\\s+$", "").replaceAll("\\s+", " ").split(" "))
            .map(Integer::parseInt)
            .collect(Collectors.toList());

        int result = Result.birthdayCakeCandles(candles);

        System.out.println("Result: " + result);

        in.close();
    }
}

class Result {
    public static int birthdayCakeCandles(List<Integer> candles) {
        int max = Collections.max(candles);
        int cont=0;

        for(int e: candles)
            if(e == max) cont++;

        return cont;
    }
}

File: 9_BirthdayCakeCandles/birthdayCakeCandles.py
def birthdayCakeCandles(candles: [int]) -> int:
    maximum = max(candles)
    cont = 0

    for e in candles:
        if e == maximum:
            cont += 1

    return cont

if __name__ == '__main__':
    n = int(input("Insert the number of candles: "))
    candles = list(map(int, input("Insert the candles: ").rstrip().split()))

    result = birthdayCakeCandles(candles)
    print(result)
